apps/emails-and-messages/src/modules/mjml/configuration/mjml-config-input-schema.ts

```diff
@@ -2,34 +2,6 @@ import { z } from "zod";
 import { messageEventTypes } from "../../event-handlers/message-event-types";
 import { smtpEncryptionTypes } from "./mjml-config";
 
-export const mjmlConfigInputSchema = z.object({
-  configurations: z.array(
-    z.object({
-      active: z.boolean(),
-      configurationName: z.string(),
-      senderName: z.string(),
-      senderEmail: z.string().email(),
-      smtpHost: z.string(),
-      smtpPort: z.string(),
-      smtpUser: z.string().min(0),
-      useTls: z.boolean(),
-      useSsl: z.boolean(),
-      templateInvoiceSentSubject: z.string(),
-      templateInvoiceSentTemplate: z.string(),
-      templateOrderCancelledSubject: z.string(),
-      templateOrderCancelledTemplate: z.string(),
-      templateOrderConfirmedSubject: z.string(),
-      templateOrderConfirmedTemplate: z.string(),
-      templateOrderFullyPaidSubject: z.string(),
-      templateOrderFullyPaidTemplate: z.string(),
-      templateOrderCreatedSubject: z.string(),
-      templateOrderCreatedTemplate: z.string(),
-      templateOrderFulfilledSubject: z.string(),
-      templateOrderFulfilledTemplate: z.string(),
-    })
-  ),
-});
-
```

Comment:
Found old version of the configuration input which is no longer used by the tRPC. Since it was exported, had no notification in the editor that it can be removed and missed it during the last refactor

Reply:
okay, will be good if we one day figure out some "unused symbols" analysis on PR. we can exclude pages, configs etc and rest could be highlighted